pori_python/ipr/annotate.py

```diff
@@ -122,9 +125,16 @@ def annotate_expression_variants(
             continue
         try:
             matches = gkb_match.match_expression_variant(graphkb_conn, gene, variant)
-            for ipr_row in get_ipr_statements_from_variants(graphkb_conn, matches, disease_name):
-                ipr_row["variant"] = row["key"]
-                ipr_row["variantType"] = row.get("variantType", "exp")
+            for matched_stmt in get_ipr_statements_from_variants(
+                graphkb_conn, matches, disease_name
+            ):
+                ipr_row = {
+                    "variant": row["key"],
+                    "variantType": row.get("variantType", "exp"),
+                    "kbVariantId": matched_stmt["kbVariantId"],
+                    "kbVariant": matched_stmt["kbVariant"],
+                    "kbMatchedStatements": [matched_stmt],
```

Comment:
it is at the time of this code, yes, because that's what the output of get_ipr_statements_from_variants is. might make more sense to put some of this in that func actually. working on it.

Reply:
moved it upstream